src/utils/index.ts

```diff
@@ -1,4 +1,5 @@
-export * from './constants.js'
+export type * from 'eslint-import-context'
```

Comment:
Should we need to export the types here? Shouldn't the types be directly imported from `eslint-import-context`?

Reply:
For compatibility.

Reply:
At least we need to restrict ourselves from referencing those types.

Let's configure the ESLint rule to do so: https://typescript-eslint.io/rules/no-restricted-imports